content/en/docs/installation/authentication.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Authentication"
+weight: 40
+description: >
+  Guide to set up basic authentication with Fission
+---
+
+## Authentication in Fission
+
+When using Fission, if you are using an ingress, you might already have some form of authentication in place for external calls.
+But if you aren't, Fission didn't provide a way to configure authentication for your API calls.
+
+Starting with v1.16.0, Fission allows you to have an authentication mechanism in place.
+
+## Understanding Authentication in Fission
+
+Fission does so by enabling authentication for Fission Router.
+This is an optional feature that can be enabled/disable depending on your requirement.
+
+When enabled, a new endpoint for authentication will be registered in the router.
+All the API calls to Fission functions will now be routed through this endpoint.
+
+Fission will also create a secret named `router` in `fission` namespace with a default `username`, a randomly generated `password` and a `jwtSigninKey`.
+This secret is mounted on a volume on the router pod.
+The user has to first create an `auth token` by providing the `username` and `password`.
+The token generated has to be passed in the authentication header for all subsequent API calls to the function.
+
+## Enabling Authentication
+
+To enable authentication, you need to set the key `authentication.enabled` to `true`.
+This can be found in `charts/fission-all/values.yaml`.
+
+```bash
+--set authentication.enabled=true
+```
+
```

Comment:
We should also highlight how different options can be utilized along with enabling authentication.

https://artifacthub.io/packages/helm/fission-charts/fission-all?modal=values&path=authentication